drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -457,11 +500,12 @@ def _is_finished(self):
         completed_count = 0
         total_count = 0
         item_state = self._get_item_state()
+
         for item in self.data['items']:
-            if item['zone'] != 'none':
+            item_id = item['id']
+            if self._get_valid_item_zones(item_id) != []:
                 total_count += 1
-                item_id = str(item['id'])
-                if item_id in item_state:
+                if str(item_id) in item_state:
                     completed_count += 1
```

Comment:
@haikuginger Hm, I don't think that grade calculation will change in assessment mode (more specifically, *how* grades are calculated should stay the same; *when* they are calculated will change), and standard mode will keep using the current version of `_is_finished`. Are there any specific changes that you think would make it necessary to touch a `_get_item_stats` method again if we implemented it now?